section2/reflection.md

```diff
@@ -1,15 +1,30 @@
+
 3. In this section, we removed some of the supports that you had in Section 1. We didn't give the directions for how to run a file in node and made some directions a bit more vague, intentionally. How did that feel? What was uncomfortable about it? How did it support your learning?
 
+I felt very comfortable with it. In the last section I had issues running the copy and pasted node command so I figured out how it worked then. That made it real simple for this time.
+
 4. What is a conditional statement? Give one example of a daily life conditional. Give one example of where a conditional is probably used in a web application you use.
 
+A condition statement is a set of instructions to evaluate variables and choose an outcome based on what the operator instructions are. A conditional statement could be as simple as breakfast. Condition statement would be, if fruit > 0 I will make a parfait. if else if bagels > 0 I will make bagel with cream cheese. For a website lets use REI and discounts. When a user returns to the website it checks their token and validates if this previous user is a member. "If" the user is a member, then show deals and discounts for members only. "If else" this user is not a member, show regular products and no special deals.
+
 5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do.
 
+I would use logical operators to make sure that both conditions are meet with either true's or false's to then print the correct log to the console. The && logical operator only produces a True result when both conditions are True and the inverse of that is || producing a false result only when both conditions are false. I saw that there are two more logical operators ! and ?? which i need to look into.
+
 6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?
 
-7. What questions do you still have about `if` statements and/or functions?
\ No newline at end of file
+You can use logical operators to make sure that you are receiving the expected output. You can also adjust the variable values to ensure the output is whats expected at these different values.
+
+7. What questions do you still have about `if` statements and/or functions?
+
+Is there a limit to how many conditional you can apply with logical operators. for example (" " && " " && " "), could this have as many assigned variables as I want? Also will we receive any exercises on switch statements. If not ill take the time to go practice those.
```

Comment:
Great reflections! Keep up the good work.

You can "chain" conditionals, but it's generally good practice to keep things as simple as possible to reduce the "cognitive load" for someone trying to read and understand your code. You'll learn more best practices about this in Mod 1 and beyond!

Switch statements are a great tool to do that^, and you'll get some exposure, but most focus will be on if-else as you build your foundations!